### PR TITLE
Constructor Drone Second Trait

### PR DESCRIPTION
**Pull request by dfuchs2**

### CODE REVIEW

File: CharacterCreation/01_02_Species_ECSC_Security_Drone.txt [hunk 1]
```diff
@@ -229,3 +229,11 @@ drones, to deal with the constant heavyweight lifting that is required of them.
     
 Size: Large
 Favored Terrain: Urban
+
+Engineering-Minded:
+    The Series Twelve Drone is programmed to better understand structural stability and 
+weaknesses. Knowledge is powerful, and Constructor Series Drones know exactly where to 
+apply force to do the most damage. Gain +20 when trying to weaken or demolish structures 
+such as walls or doors. Constructor Drones are built to be very resourceful when it comes
+to building. Gain Jury-Rigging at 15 if you take it as a level 1 skill, or at 20 if you
```
Comment (bleehu) [inline CharacterCreation/01_02_Species_ECSC_Security_Drone.txt]: 15 and 20 are a little too high for level 1 with @rosebrallier's new skill system, where 6 or 12 would be the max expected of a level 1 character. Also, Engineers already get a flat bonus of 10 to Jury-rigging as well as proficiency in the skill. I really like the idea that construction bots are better at learning to build things, so I hope we keep the theme. I really like it. Rev suggests changing this to advantage on jury-rigging instead?

**Review by bleehu — CHANGES_REQUESTED** [review]
This is really good, but a bit OP. I think the obvious synergy between Construction bot as a species and engineer as a class has been telegraphed well by their names. But we'll need to nerf the bonus and/or spread the benefit around a bit because this turns into like a 30-something jury rigging at level 1, which is higher than we want.

**Review by Turtlelord26 — COMMENTED** [review]
This one also looks like two traits. Most of the traits currently in place are fairly narrow, which allows more customization when building characters.

I would suggest (again) to address blee's first concern by swapping out the flat bonuses to Advantage rolls. Advantage is pretty good tech because it's less powerful than +20 but feels so much cooler. Advantage will also be more powerful in the late game because, unlike giving a bunch of points in jury-rig at level 1, it won't count against the 60 skill point cap.

**Review by bleehu — APPROVED** [review]
Shipit!

**Comment by bleehu** [comment]
Linking Issue #555

**Pull request merged**